plugins.txt

```diff
@@ -1,21 +1,23 @@
-ansicolor:0.7.3
+ansible:1.1
+ansicolor:0.7.5
 badge:1.8
-blueocean:1.24.3
-configuration-as-code:1.46
-docker-workflow:1.25
-github-branch-source:2.9.2
+blueocean:1.24.5
+configuration-as-code:1.47
+docker-workflow:1.26
+github-branch-source:2.10.2
 greenballs:1.15.1
 job-dsl:1.77
-kubernetes:1.25.7
-parameterized-trigger:2.39
+kubernetes:1.29.2
```

Comment:
Careful here, that was the plugin that breaks before

Reply:
Has critical security issues, I hope the double output is not there anymore but I think it overrules cosmetics anyways.

Reply:
Seems like it's fixed:
https://jenkins.oss-eks.dev.emnify.io/job/test-debug-ansible-pipeline/4/console
VS
https://jenkins.oss-eks.dev.emnify.io/job/test-debug-ansible-pipeline/5/console